### PR TITLE
Beyond the Basics - Public Attestation (clean commits)

### PR DESCRIPTION
**Pull request by CamrynCarter**
PR for the Public Attestation documentation, replacing https://github.com/jitsuin-inc/rkvst-docs/pull/363 so the commits are clean and easier to review.

### CODE REVIEW

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 1]
```diff
@@ -1,14 +1,175 @@
...
      parent: "beyond-the-basics"
- weight: 41
- toc: false
----
\ No newline at end of file
+ weight: 42
+ toc: true
+---
+
+You may wish to attest information to the general public, without the need for viewers to log-in to their RKVST account. `Public Assets` can be used to publicly assert data, also referred to as Public Attestation. For example, the vulnerability reports against an OpenSource software package or the maintenance records for a building.
```
Comment (kjc-jitsuin) [inline content/docs/beyond-the-basics/public-attestation/index.md:16]: "For example, the vulnerability reports against an OpenSource software package or the maintenance records for a building."

Can we say these are things you may wish to attest to? 

Propose "For example, you may attest to data containing a vulnerability report against an OpenSource software package or maintanence records for a building."

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 2]
```diff
@@ -1,14 +1,175 @@
...
+    https://app.rkvst.io/archivist/v2/assets
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+3. Retrieve public link to share your Public Asset with others. 
+
+{{< note >}}
+**NOTE:** Public Assets may only be updated by the creating tenancy. Anyone viewing the Asset using the public link will have read-only access.
```
Comment (kjc-jitsuin) [inline content/docs/beyond-the-basics/public-attestation/index.md:95]: Is "creating tenancy" used elsewhere in the docs? Is there a better wording for this? Asset owning tenancy?

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 1 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:16]: Change implemented

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 2 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:95]: Changed to "A Public Asset may only be updated by the tenancy that created it. Anyone viewing the Asset using the public link will have read-only access" so it is more clear, as "creating tenancy" isn't used elsewhere.

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 3]
```diff
@@ -1,14 +1,175 @@
...
- toc: false
----
\ No newline at end of file
+ weight: 42
+ toc: true
+---
+
+You may wish to attest information to the general public, without the need for viewers to log-in to their RKVST account. `Public Assets` can be used to publicly assert data, also referred to as Public Attestation. For example, you may attest to data containing a vulnerability report against an OpenSource software package or the maintenance records for a building.
+
+Private Assets can only be shared through the creation of [Access Policies](../../rkvst-basics/sharing-assets-with-obac/). Public Assets, however, may be shared with a read-only `Public URL`, similar to the link sharing you may have seen in file sharing services such as Google Drive or DropBox.
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:18]: Minor: I think there was alot of discussion about what to call a non-public asset, I remember there being some pushback about the use of `private asset` it maybe worth a discussion with the wider team to agree on a name for non-public asset?

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 4]
```diff
@@ -1,14 +1,175 @@
...
+You may wish to attest information to the general public, without the need for viewers to log-in to their RKVST account. `Public Assets` can be used to publicly assert data, also referred to as Public Attestation. For example, you may attest to data containing a vulnerability report against an OpenSource software package or the maintenance records for a building.
+
+Private Assets can only be shared through the creation of [Access Policies](../../rkvst-basics/sharing-assets-with-obac/). Public Assets, however, may be shared with a read-only `Public URL`, similar to the link sharing you may have seen in file sharing services such as Google Drive or DropBox. 
+
+Any Events updating a Public Asset will also be public, and will each have their own unique Public URL.
+
+Following the link to a Public Asset or Public Event will allow read-only access to its information, without the need to sign in to RKVST.
+
+
+## Creating a Publicly Attested Asset
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:25]: It maybe useful to add a reference in `RKVST Basics -> Creating An Asset` pointing here if someone wants to create a public asset?

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 5]
```diff
@@ -1,14 +1,175 @@
...
+
+{{< warning >}}
+**Warning**: Assets can only be made public at Asset Creation and cannot be made private afterwards.
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:28]: Minor: I think we need to make it really clear here that the asset will remain public forever, including any events associated with the asset?

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 6]
```diff
@@ -1,14 +1,175 @@
...
+{{{< tab name="UI" >}}
+Select `Add Asset` from the sidebar and fill in the desired details.
+{{< img src="CreateAsset.png" alt="Rectangle" caption="<em>Asset Details</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Create a JSON file with your desired Asset details. 
+
+```json
+{
+    "behaviours": ["RecordEvidence", "Attachments"],
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:43]: Minor: I think splitting out this step and the next step for json is slightly confusing? As a developer I would prefer to have the entire example json payload in one step

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 7]
```diff
@@ -1,14 +1,175 @@
...
+    https://app.rkvst.io/archivist/v2/assets
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+3. Retrieve public link to share your Public Asset with others. 
+
+{{< note >}}
+**NOTE:** A Public Asset may only be updated by the tenancy that created it. Anyone viewing the Asset using the public link will have read-only access.
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:95]: nice!

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 8]
```diff
@@ -1,14 +1,175 @@
...
+{{< /tabs >}}
+
+4. The following screenshot shows the public view of the Asset when the link is followed. 
+{{< img src="PublicView.png" alt="Rectangle" caption="<em>Public View</em>" class="border-0" >}}
+
+
+### Adding an Event to a Public Asset
+
+{{< note >}}
+**NOTE:** Any Events added to a Public Asset will also be public. Events may only be added by the tenancy that originally created the Public Asset.
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:121]: really nice and clear :)

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 3 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:18]: Yes good point, I will add that to the discussion topics for the documentation meeting next week.

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 4 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:25]: I linked to that section in step one, "1. Create an Asset with your desired attributes. See [Creating an Asset](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/) for detailed instructions." but let me know if you think it needs to be more clear.

Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:25]: Good idea, I will add that!

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 5 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:28]: Changed to: " Once an Asset is made public, it cannot be made private. Any information published will be publicly accessible forever."

Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:28]: Change reflected in both warnings to make sure it is clear.

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 6 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:43]: The initial reason was to follow the steps (1. create asset, 2. make it public), but I agree it's slightly confusing. I will change to having the private keyword in the initial asset creation step and change step two to publish asset.

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 3 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md]: Term 'private assets' replaced by 'permissioned assets'.

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 9]
```diff
@@ -1,14 +1,165 @@
...
+
+Any Events updating a Public Asset will also be public, and will each have their own unique Public URL.
+
+Following the link to a Public Asset or Public Event will allow read-only access to its information, without the need to sign in to RKVST.
+
+
+## Creating a Publicly Attested Asset
+
+{{< warning >}}
+**Warning**: Assets can only be made public at Asset Creation and cannot be made private afterwards. Any information published will be publicly accessible forever.
```
Comment (henry739) [inline content/docs/beyond-the-basics/public-attestation/index.md:28]: Its worth adding here that events created on a publicly attested asset are also publicly viewable forever. Perhaps the second sentence can become:

"This asset and all its events will be publicly accessible forever."

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 10]
```diff
@@ -1,14 +1,165 @@
...
+{{< /warning >}}
+
+1. Create an Asset with your desired attributes and set it to public. See [Creating an Asset](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/) for detailed instructions. 
+
+{{< tabs name="create_asset_public" >}}
+{{{< tab name="UI" >}}
+Select `Add Asset` from the sidebar and fill in the desired details.
+{{< img src="CreateAsset.png" alt="Rectangle" caption="<em>Asset Details</em>" class="border-0" >}}
+Check the box next to `Make Asset Public`.
```
Comment (henry739) [inline content/docs/beyond-the-basics/public-attestation/index.md:37]: For some reason this isn't rendering correctly on my machine:

image

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 11]
```diff
@@ -1,14 +1,165 @@
...
+     https://app.rkvst.io/archivist/v2/assets/<asset-id>:publicurl
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+4. The following screenshot shows the public view of the Asset when the link is followed. 
+{{< img src="PublicView.png" alt="Rectangle" caption="<em>Public View</em>" class="border-0" >}}
+
+
+### Adding an Event to a Public Asset
```
Comment (henry739) [inline content/docs/beyond-the-basics/public-attestation/index.md:108]: Question: AFAIK creating an event against a public asset (when logged in as the owning tenant) is the same as for a normal, permissioned asset (which is documented in "Creating an Event Against an Asset" as you pointed out in step 1.

Is duplicating this information mostly a convenience thing for readers?

File: content/docs/rkvst-basics/creating-an-asset/index.md [hunk 12]
```diff
@@ -18,7 +18,9 @@ aliases:
 
 An Asset can be anything: a Connected Machine, a Shipping Container, or even a Data Set. It can be any physical or digital object with an associated Name, Description, and Attributes.
 
-Each Asset will have a history of any actions performed upon it by any actor. 
+Each Asset will have a history of any actions performed upon it by any actor.
+
+You may wish to publicly attest an Asset's history. For more information on how to do so, see [Public Attestation](https://docs.rkvst.com/docs/beyond-the-basics/public-attestation/).
```
Comment (henry739) [inline content/docs/rkvst-basics/creating-an-asset/index.md:23]: Minor: I (as a non-docs person) might rephrase this as: "RKVST enables you to publicly attest to the provenance of your Assets. To learn how, see {Link}"

**Review by henry739 — COMMENTED** [review]
This update looks great! I've got a few comments, mostly minor/stylistic points. Please let me know if you have any questions or want a re-review/tick.

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 13]
```diff
@@ -1,14 +1,165 @@
...
- toc: false
----
\ No newline at end of file
+ weight: 42
+ toc: true
+---
+
+You may wish to attest information to the general public, without the need for viewers to log-in to their RKVST account. `Public Assets` can be used to publicly assert data, also referred to as Public Attestation. For example, you may attest to data containing a vulnerability report against an OpenSource software package or the maintenance records for a building.
+
+Permissioned Assets can only be shared through the creation of [Access Policies](../../rkvst-basics/sharing-assets-with-obac/). Public Assets, however, may be shared with a read-only `Public URL`, similar to the link sharing you may have seen in file sharing services such as Google Drive or DropBox.
```
[the remark this answers is not in the document]
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:18]: Wording changed to 'a Public URL that points to a read-only view of the Asset'.

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 9 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:28]: I agree, change made on both warnings.

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 10 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:37]: Thanks for catching, there was a spacing error in the markdown that caused the break. Fixed!

File: content/docs/beyond-the-basics/public-attestation/index.md (hunk 11 above)
Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:108]: Yes, this is just a continuing example for readers to follow. It also allows us to show that the event on a public asset is automatically listed in public view. I think the more clear/explicit we can make the information, the better; especially with public assets/events being forever accessible after they're published.

File: content/docs/rkvst-basics/creating-an-asset/index.md (hunk 12 above)
Comment (CamrynCarter) [reply content/docs/rkvst-basics/creating-an-asset/index.md:23]: I prefer that wording as well, I changed the paragraph to note the options for permissioned sharing or public attestation.

File: content/docs/beyond-the-basics/public-attestation/index.md [hunk 14]
```diff
@@ -1,14 +1,175 @@
 ---
- title: "Public Assets vs Private Assets"
- description: "Transparency through Public Attestation"
+ title: "Public Attestation"
+ description: "Public Assets vs Private Assets"
```
Comment (honourfish) [inline content/docs/beyond-the-basics/public-attestation/index.md:3]: This should now be ` vs permissioned assets`?

Comment (CamrynCarter) [reply content/docs/beyond-the-basics/public-attestation/index.md:3]: Yep, thanks for catching!

**Review by kjc-jitsuin — APPROVED** [review]

**Pull request merged**